bin/wpc_init

```diff
@@ -121,7 +134,22 @@ set -e
 ## This code will be run during setup, INSIDE the container.
 ##
 
-wp core install --skip-email --admin_user=admin --admin_password=admin --admin_email=admin@localhost.invalid --url=http://localhost --title=Site
-# wp theme activate ...
-# wp plugin activate ...
+##############
+# Config
+##############
+title=your_site_title
+theme=your-theme-slug
+plugins="a-space-separated list-of plugins-to-activate"
+content=/usr/src/app/setup/content/*
+
+wp core install --skip-email --admin_user=admin --admin_password=admin --admin_email=admin@localhost.invalid --url=http://localhost --title=$title
+wp plugin activate $plugins
+wp theme activate $theme
```

Comment:
By trying to activate themes and plugins that don't exist, we're ensuring that `./bin/setup` will produce an error unless modified.

This means that somebody using wpc who's maybe being a little careless, could run `wpc_init`, add the files to git, and commit without modifying `setup/internal.sh`. This might happen if they just need to add the docker-compose file bit, but they've already got a copy of the site up and running already.

Could we add some `if` statements here to catch that scenario?